### DISCUSSION TITLE
Problems recognizing digital digits between the number 1 and 7

### DISCUSSION THREAD

**Discussion by lauer** [jomjol/AI-on-the-edge-device#3121] [Q&A]
I have an issue, that one of my 1 digits is often recognized as a 7. 

I have tried to adjust the roi areas, so they are a bit smaller at the last 3 digits, but strange enough, the dec1 is detected as 7 where the dec3 is detected correctly as 1.
![digits_processing](https://github.com/jomjol/AI-on-the-edge-device/assets/2368/c6af1327-bfd9-45e7-bd2c-cc77ef15d628)

I am using the dig-class11_1500_s2 model.

You can see my setup here (older reference image)
![digits_roi_setup](https://github.com/jomjol/AI-on-the-edge-device/assets/2368/a3354cb9-25da-4976-9d4d-f95db0d23393)

Can I do something to approve this, or should I try to train my own model, like described here?
https://jomjol.github.io/AI-on-the-edge-device-docs/Learn-models-with-your-own-images/

**Comment by ghost** [comment]
I would try to set the ROIs so that the border ist not in the roi.

It may help collecting images or to try different models

**Reply by SybexX** [reply]
https://github.com/jomjol/AI-on-the-edge-device/issues/3144#issuecomment-2241481520
